README.md

```diff
@@ -23,24 +23,74 @@ To use the driver, it can be installed using NuGet package manager. The driver p
 
 ```Amazon.QLDB.Driver```
 
+You will also need to add the Amazon
```

Comment:
This line is little ambiguous. Do you mean to say "Make sure you prepend Amazon?

Reply:
wil fix.